packs/data/feats.db/elemental-embellish.json

```
@@ -13,19 +13,26 @@
             "value": null
         },
         "description": {
-            "value": "<p>You can summon a harmless but impressive elemental display. You become trained in Intimidation. If you would automatically become trained in Intimidation (from your background or class, for example), you instead become trained in a skill of your choice. When you Demoralize a foe, if you choose to Demoralize via an elemental display, Demoralize loses the auditory trait and gains the visual trait, and you don't take a penalty when you attempt to Demoralize a creature that doesn't understand your language.</p>"
+            "value": "<p>You can summon a harmless but impressive elemental display. You become trained in Intimidation. If you would automatically become trained in Intimidation (from your background or class, for example), you instead become trained in a skill of your choice.</p>\n<p>When you Demoralize a foe, if you choose to Demoralize via an elemental display, Demoralize loses the auditory trait and gains the visual trait, and you don't take a penalty when you attempt to Demoralize a creature that doesn't understand your language.</p>"
```

```suggestion
            "value": "<p>You can summon a harmless but impressive elemental display. You become trained in Intimidation. If you would automatically become trained in Intimidation (from your background or class, for example), you instead become trained in a skill of your choice. When you Demoralize a foe, if you choose to Demoralize via an elemental display, Demoralize loses the auditory trait and gains the visual trait, and you don't take a penalty when you attempt to Demoralize a creature that doesn't understand your language.</p>"
```
No new paragraph in the PDF.